docs/theming/advanced.md

```diff
@@ -32,9 +32,9 @@ The example below demonstrates how to use `theme-color` to style the browser int
 <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#eb445a" />
 ```
 
-| Light Mode                                                                             | Dark Mode                                                                            |
-| -------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
-| ![Application with theme-color meta in light mode](/img/theming/theme-color-light.png) | ![Application with theme-color meta in dark mode](/img/theming/theme-color-dark.png) |
+| Light Mode                                                                                                                                  | Dark Mode                                                                                                                                |
+| ------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| ![Screenshot demonstrating the light mode theme color in a browser interface.](/img/theming/theme-color-light.png 'Light Mode Theme Color') | ![Screenshot demonstrating the dark mode theme color in a browser interface.](/img/theming/theme-color-dark.png 'Dark Mode Theme Color') |
```

Comment:
This one also needs to updated to exclude "screenshot". The same goes for the same sentences found in other files.